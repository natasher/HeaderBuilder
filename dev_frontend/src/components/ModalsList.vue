<template>
  <section name="modals">
    <mfn-generic-modal  />
    <modal-button       />
    <modal-grid         />
    <modal-action-bar   />
    <modal-first-row    />
    <modal-second-row   />
    <modal-extras       />
    <modal-icon         />
    <modal-image        />
    <modal-logo         />
    <modal-menu         />
    <modal-menu-icon    />
    <modal-social-media />
    <modal-text         />
  </section>
</template>

<script>
import MfnGenericModal  from './Forms/MfnGenericModal.vue';
import ModalButton      from './Forms/Modals/ModalButton.vue';
import ModalGrid        from './Forms/Modals/ModalGrid.vue';
import ModalActionBar   from './Forms/Modals/ModalActionBar.vue';
import ModalFirstRow    from './Forms/Modals/ModalFirstRow.vue';
import ModalSecondRow   from './Forms/Modals/ModalSecondRow.vue';
import ModalExtras      from './Forms/Modals/ModalExtras.vue';
import ModalIcon        from './Forms/Modals/ModalIcon.vue';
// TODO: change name ModalImage to different name (Picture??), non html related, it couses warn
import ModalImage       from './Forms/Modals/ModalImage.vue';
import ModalLogo        from './Forms/Modals/ModalLogo.vue';
import ModalMenu        from './Forms/Modals/ModalMenu.vue';
import ModalMenuIcon    from './Forms/Modals/ModalMenuIcon.vue';
import ModalSocialMedia from './Forms/Modals/ModalSocialMedia.vue';
import ModalText        from './Forms/Modals/ModalText.vue';

export default {

  name: 'ModalsList',

  components: {
    MfnGenericModal,
    ModalButton,
    ModalGrid,
    ModalActionBar,
    ModalFirstRow,
    ModalSecondRow,
    ModalExtras,
    ModalIcon,
    ModalImage,
    ModalLogo,
    ModalMenu,
    ModalMenuIcon,
    ModalSocialMedia,
    ModalText,
  },

}
</script>
