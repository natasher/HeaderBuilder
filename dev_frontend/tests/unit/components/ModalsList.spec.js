import { shallowMount } from '@vue/test-utils'
import ModalsList from '../../../src/components/ModalsList'

describe('ModalList.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount(ModalsList)

    expect( wrapper.element ).toMatchSnapshot()
  })

})