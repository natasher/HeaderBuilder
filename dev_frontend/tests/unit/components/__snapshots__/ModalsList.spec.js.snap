// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalList.vue should match snapshot 1`] = `
<section
  name="modals"
>
  <mfngenericmodal-stub />
   
  <modalbutton-stub />
   
  <modalgrid-stub />
   
  <modalactionbar-stub />
   
  <modalfirstrow-stub />
   
  <modalsecondrow-stub />
   
  <modalextras-stub />
   
  <modalicon-stub />
   
  <modalimage-stub />
   
  <modallogo-stub />
   
  <modalmenu-stub />
   
  <modalmenuicon-stub />
   
  <modalsocialmedia-stub />
   
  <modaltext-stub />
</section>
`;
